### ISSUE TITLE
Provide scripts for converting to healpix

### ISSUE THREAD

**Issue by florianziemen** [digital-earths-global-hackathon/tools#3]
Possible contributions by

@florianziemen @Zarnosch @lkluft , Orhan and others...

**Comment by florianziemen** [comment]
@antarcticrainforest has scripts, @feng045 would be one of the users who need to convert data.

**Comment by feng045** [comment]
Yes, we have a new version of 1+ year of SCREAM simulation covering the DYAMOND3 period that I want to convert to HEALPix using the newer workflow. @antarcticrainforest could you please point me to the example scripts that I can try?  Thanks!
